src/StagedProposalProcessor.sol

```diff
@@ -557,21 +541,141 @@ contract StagedProposalProcessor is ProposalUpgradeable, PluginUUPSUpgradeable {
             } catch {
                 // Handles the edge case where:
                 // on success: it could return 0
-                // on failure: default 0 would be used 
+                // on failure: default 0 would be used
                 // In order to differentiate, we store `uint256.max` on failure.
-                pluginProposalIds[_proposalId][_stageId][stage.plugins[i].pluginAddress] = type(
-                    uint256
-                ).max;
 
                 uint256 gasAfter = gasleft();
 
                 if (gasAfter < gasBefore / 64) {
                     revert Errors.InsufficientGas();
                 }
+
+                pluginProposalIds[_proposalId][_stageId][stage.plugins[i].pluginAddress] = type(
+                    uint256
+                ).max;
```

Comment:
```js
// Handles the edge case where:
// on success: it could return 0
// on failure: default 0 would be used
// In order to differentiate, we store `uint256.max` on failure.
```

Reply:
So, in the context of a new stage, we want to track that a plugin didn't allow creating the sub proposal?

In such, case I'd use the constant mentioned elsewhere but with this name:  `PROPOSAL_WITHOUT_ID`

Reply:
done in https://github.com/aragon/staged-proposal-processor-plugin/pull/8/commits/a404de74b5fdf3acb4f318fd58748671f12ada04